docker-compose-local.yml

```diff
@@ -19,12 +19,19 @@ services:
       - PORT=1212
       - KUBE_SCHEDULER_SIMULATOR_ETCD_URL=http://simulator-cluster:2379
       - KUBE_APISERVER_URL=http://simulator-cluster:3131
+    volumes:
+      - ./simulator/config.yaml:/config.yaml
```

Comment:
Given we modify `Dockerfile` itself, please add the same fix for `/config.yaml` to docker-compose.yaml too.